test/e2eTests/tests/deletePatient_test.js

```diff
@@ -0,0 +1,140 @@
+const assert = require('assert');
+
+// Test for:
+// https://docs.google.com/document/d/1tav6VO5tl2ghL-owOgKSV4B-w3LhGZo4NyUCAbBxhIE/edit#
+
+Feature('Delete Patient');
+
+const LOG_TAG = '[DeletePatientTests]';
+
+let Patient = null;
+
+Before(async (I, Apis, Data) => {
+  I.say(`${LOG_TAG} Creating patient to be deleted`);
+  Patient = await Apis.patient.create(Data.patients.patient2);
+});
+
+// TODO: Centralize this to be reused
```

Comment:
Might as well do this in these changes, right? Maybe we should have a utils class. I'm thinking /tests/e2eTests/utils.js

export default class Utils {
    static cleanUpPatientVisit(patient) { ... }
}

Or something like that

Reply:
And the other functions, of course

Reply:
I'm reviewing the patient registration next and I'll include the utils class.